db/config.cc

```diff
@@ -715,7 +715,6 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , replace_address_first_boot(this, "replace_address_first_boot", value_status::Used, "", "Like replace_address option, but if the node has been bootstrapped successfully it will be ignored. Same as -Dcassandra.replace_address_first_boot.")
     , override_decommission(this, "override_decommission", value_status::Used, false, "Set true to force a decommissioned node to join the cluster")
     , enable_repair_based_node_ops(this, "enable_repair_based_node_ops", liveness::LiveUpdate, value_status::Used, false, "Set true to use enable repair based node operations instead of streaming based")
-    , wait_for_hint_replay_before_repair(this, "wait_for_hint_replay_before_repair", liveness::LiveUpdate, value_status::Used, true, "If set to true, the cluster will first wait until the cluster sends its hints towards the nodes participating in repair before proceeding with the repair itself. This reduces the amount of data needed to be transferred during repair.")
     , ring_delay_ms(this, "ring_delay_ms", value_status::Used, 30 * 1000, "Time a node waits to hear from other nodes before joining the ring in milliseconds. Same as -Dcassandra.ring_delay_ms in cassandra.")
```

Comment:
This changes a user-facing API. So the changelog has to explain why it is safe.

Reply:
Information added to the commit description (the option becomes useless and it's not a part of any release)